lib/fastlane/actions/carthage.rb

```diff
@@ -26,16 +21,13 @@ def self.description
 
       def self.available_options
         [
-          FastlaneCore::ConfigItem.new(key: :update,
```

Comment:
by getting rid of this option we are going to break users that are using this in their Fastfiles. i think a nicer option would be to add optional options for `build` and `bootstrap` as there are only three commands to be used, it would be better IMO to be explicit about them instead of allow a user to enter any arbitrary string that may fail.